### PR TITLE
Forms Automated Test - LPS-133471 | master

### PR DESCRIPTION
**Pull request by Jamersonze**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 1]
```diff
@@ -651,6 +651,13 @@ definition {
 			radioOption = "${textFieldLineOption}");
 	}
 
+	macro viewAnyField {
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro]: Good job, @Jamersonze! Let's just rename this macro and the associated path removing the word **any**, e.g. let's rename to `viewField` and `FormFields#FIELD`.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
@Jamersonze, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?

Thanks.

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro (hunk 1 above)
Comment (Jamersonze) [reply portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro]: Renamed

**Comment by Jamersonze** [comment]
Fixed

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**